### PR TITLE
Reset PLATFORM enviroment variable in build.bat

### PR DESCRIPTION
**Pull request by un-tone**

### CODE REVIEW

**Comment by warpech** [comment]
Thank you for the investigation @un-tone. Since we want to replace the build script with a simpler one, maybe it's better to replace the script already?

```txt
@ECHO OFF

:: Set up the env to use Msbuild 14.0
CALL "%VS140COMNTOOLS%\vsvars32.bat"

:: Try to restore packages (comment this out if not needed)
WHERE nuget.exe >nul 2>nul
IF %ERRORLEVEL% EQU 0 nuget.exe restore
IF NOT EXIST "packages\" (ECHO Error: Get nuget.exe or build the sln in VS to restore the packages && EXIT /B 1)

PUSHD %~dp0
msbuild
POPD
```

See: https://github.com/StarcounterPrefabs/Guidelines/issues/6#issuecomment-274775969

**Comment by un-tone** [comment]
@warpech Done.

File: build.bat [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
-
-:END
+:: Set up the env to use Msbuild 14.0
+CALL "%VS140COMNTOOLS%\vsvars32.bat"
+SET PLATFORM=
 
+:: Try to restore packages (comment this out if not needed)
+WHERE nuget.exe >nul 2>nul
+IF %ERRORLEVEL% EQU 0 nuget.exe restore
+IF NOT EXIST "packages\" (ECHO Error: Get nuget.exe or build the sln in VS to restore the packages && EXIT /B 1)
```
Comment (warpech) [inline build.bat]: Actually it seems to me that Launcher is a case where the lines 8-10 should be commented out

Comment (un-tone) [inline build.bat]: Launcher_AcceptanceTest project included in the solution uses NuGet packages

Comment (warpech) [inline build.bat]: True, sorry!

**Pull request merged**